tests/ui/specs/Picker/Picker-specs.js

```diff
@@ -0,0 +1,102 @@
+const Page = require('./PickerPage');
+const {extractValue} = require('./Picker-utils.js');
+
+describe('Picker', function () {
+
+	describe('LTR locale', function () {
+		beforeEach(function () {
+			Page.open();
+		});
+
+		describe('default', function () {
+			const picker = Page.components.pickerDefault;
+
+			describe('5-way', function () {
+				it('should change the value forward when incrementing the picker', function () {
+					browser.waitUntil(() => picker.decrementer(picker.self).isFocused(), {timeout: 1500,  interval: 100});
```

Comment:
I'm not sure we need this `waitUntil` for all the cases you used.
Please verify it's okay or not without this.
You could do just `expect(picker.decrementer(picker.self).isFocused()).to.be.true();`.

Reply:
alright